src/components/headings.js

```diff
@@ -83,38 +83,45 @@ const ContentSubH1 = function ({ children }) {
 */
 const IntervalColumnHeadings = function ({ type }) {
 
-  let columnTitle = type.toLowerCase().replace(/\b[a-z]/g, (letter) => {
+  let styles      = { fontSize: `14px` },
+      toUpper     = function (letter) {
         return letter.toUpperCase();
-      }) + ' Type',
-      styles      = { fontSize: '14px' };
+      },
+      asLowerCase = type.toLowerCase(),
+      toReplace   = /\b[a-z]/g,
```

Comment:
Readability. We try to limit the number of operations happening on one line and name things in a way that makes it clear what's going on. Thoughts?

Reply:
Personally, I think I found it easier to follow before--this way, I have to look around to find out what toReplace actually is, how toUpper is defined, etc. This may just be a personal preference thing; I'd be interested to hear what other people think about it. @dylanesque ?

Reply:
I love the sound of this discussion! In the meantime, are we holding off on merging for this? Or is this a discussion to have separately? Would you prefer that I change it back till we've discussed it more? Just so we can get this PR through.

Reply:
Yeah, I don't see this as blocking, just putting in my two cents.

Reply:
@todo then!